document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/data/accounts/AccountsManager.java

```diff
@@ -148,6 +148,14 @@ public SmallFullAccountIxbrl getSmallFullAccounts(String link, Transaction trans
             handleException(e, "debtors", link);
         }
 
+        try {
+            smallFullApiData.setCreditorsWithinOneYear(apiClient.smallFull().creditorsWithinOneYear()
```

Comment:
Daf's just merged a PR into develop which refactors the way in which resources are retrieved - we now iterate over the small full resource links and perform a `GET` against each endpoint. You'll need to merge `develop` into this branch and use the same mechanism to set the creditors within one year resource on `smallFullApiData`.